icon-registry/icons-static/action-upgrade-square_x16.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg" class="icon-dark">
```

Comment:
You should have this class only on the path so the system can guess if strke or path shoudl be changed.

Also, you should not use a `fillColor` attribute here (though it will not be used in the system). It makes it harder to recognize which file is which icon in the explorer.

Traditionally we also remove the `width` and `height` attributes to get a better zoom on it in macos/windows/github